docs/rst/user_manual/configuration.rst

```diff
@@ -280,40 +336,34 @@ This is a YAML file that uses all supported configurations and set them as defau
       domain: 0
 
       allowlist:
-        - name: "topic_name"
-          type: "topic_type"
+        - name: topic_name
```

Comment:
Router and recorder doc's general example include quotation marks in allowlist and blocklist elements.